bundles/core/org.eclipse.smarthome.core.thing.xml/thing-description-1.0.0.xsd

```diff
@@ -181,5 +182,43 @@
 			</xs:extension>
 		</xs:simpleContent>
 	</xs:complexType>
+
+	<xs:simpleType name="dimension">
+		<xs:restriction base="xs:string">
+			<xs:enumeration value="Acceleration" />
+			<xs:enumeration value="AmountOfSubstance" />
+			<xs:enumeration value="Angle" />
+			<xs:enumeration value="Area" />
+			<xs:enumeration value="CataliyticActivity" />

```

Comment:
I believe this is spelled "Catalytic" without the first "i"

Reply:
nice catch, thanks!

Reply:
I did correct the typo and sync the channel-type dimensions with the java enum.